package.json

```diff
@@ -71,9 +71,6 @@
       "path": "./node_modules/cz-conventional-changelog"
     }
   },
-  "eslintConfig": {
-    "extends": "react-app"
-  },
```

Comment:
I think this section was redundant with `.eslintrc.js`, and it confused me for a while, so I removed it.